easybuild/easyconfigs/p/PyYAML/PyYAML-3.12-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'PyYAML'
+version = '3.12'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://pypi.python.org/pypi/PyYAML/"
+description = """PyYAML is a YAML parser and emitter for the Python programming language."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['592766c6303207a20efc445587778322d7f73b161bd994f227adaa341ba212ab']
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('libyaml', '0.1.7', '', True),
```

Comment:
@wpoely86 Why is this hardcoded to `dummy`? We already have `libyaml-0.1.7-GCCcore-6.4.0.eb` that can resolve this dep.